./configure --without-tools is broken

regardless of the configure setup (with or without tools) when compilation of main is finished I run into `/bin/sh: line 20: cd: no: No such file or directory` Any idea where problem may lay?

```
  $ ./configure --prefix=/usr --build=x86_64-pc-linux-gnu --host=x86_64-pc-linux-gnu
 --mandir=/usr/share/man --infodir=/usr/share/info --datadir=/usr/share --sysconfdir=/etc
 --localstatedir=/var/lib --disable-dependency-tracking --disable-silent-rules
 --docdir=/usr/share/doc/quetoo-9999 --htmldir=/usr/share/doc/quetoo-9999/html 
 --libdir=/usr/lib64 --disable-debug --disable-static --without-tests --without-tools
```

```
libtool: link: x86_64-pc-linux-gnu-gcc -I../../src -pipe -march=native -O2 -pipe -DG_DISABLE_ASSERT -DNO_DEBUG -DNDEBUG -fomit-frame-pointer -ftree-vectorize -fpredictive-commoning -fgcse-after-reload -g -Wall -Wextra -Wno-unused-parameter -Wno-unknown-pragmas -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -D_REENTRANT -I/usr/include/SDL2 -pipe -march=native -O2 -pipe -DG_DISABLE_ASSERT -DNO_DEBUG -DNDEBUG -fomit-frame-pointer -ftree-vectorize -fpredictive-commoning -fgcse-after-reload -Wl,-O1 -Wl,--hash-style=both -Wl,--relax -Wl,--sort-common -o quetoo-dedicated quetoo_dedicated-main.o  -Wl,--as-needed ../../src/client/.libs/libclient_null.a ../../src/server/.libs/libserver.a -lphysfs -lz -ldl -lm -lglib-2.0 -lSDL2_image -lSDL2 -lncurses
make[3]: Leaving directory '/tmp/portage/games-fps/quetoo-9999/work/quetoo-9999/src/main'
Making all in tools
make[3]: Entering directory '/tmp/portage/games-fps/quetoo-9999/work/quetoo-9999/src/tools'
Making all in no
/bin/sh: line 20: cd: no: No such file or directory
make[3]: *** [Makefile:410: all-recursive] Error 1
make[3]: Leaving directory '/tmp/portage/games-fps/quetoo-9999/work/quetoo-9999/src/tools'
make[2]: *** [Makefile:930: all-recursive] Error 1
make[2]: Leaving directory '/tmp/portage/games-fps/quetoo-9999/work/quetoo-9999/src'
make[1]: *** [Makefile:457: all-recursive] Error 1
make[1]: Leaving directory '/tmp/portage/games-fps/quetoo-9999/work/quetoo-9999'
make: *** [Makefile:389: all] Error 2
 * ERROR: games-fps/quetoo-9999::i7woj failed (compile phase):
```

--- a/configure.ac
+++ b/configure.ac
@@ -322,10 +322,11 @@ dnl --------------------------
 
 AC_MSG_CHECKING(which tools to build)
 
-TOOLS="quemap quetoo-master quetoo-update"
+ALL_TOOLS="quemap quetoo-master quetoo-update"
+TOOLS=${ALL_TOOLS}
 
 AC_ARG_WITH(tools,
-	AS_HELP_STRING([--with-tools='quemap quetoo-master quetoo-update...'],
+	AS_HELP_STRING([--with-tools='${ALL_TOOLS}...'],
 		[build specified tools]
 	)
 )
@@ -336,6 +337,10 @@ fi
 
 if test ${TOOLS} = "no"; then
         TOOLS=""
+fi
+
+if test ${TOOLS} = "yes"; then
+        TOOLS="${ALL_TOOLS}"
 fi
 
 AC_SUBST(TOOLS)

--- a/configure.ac
+++ b/configure.ac
@@ -335,7 +335,7 @@ if test "x${with_tools}" != x; then
 fi
 
 if test ${TOOLS} = "no"; then
-        TOOLS="quetoo-master quetoo-update"
+        TOOLS=""
 fi
 
 AC_SUBST(TOOLS)

--- a/configure.ac
+++ b/configure.ac
@@ -334,6 +334,10 @@ if test "x${with_tools}" != x; then
 	TOOLS="${with_tools}"
 fi
 
+if test ${TOOLS} = "no"; then
+        TOOLS="quetoo-master quetoo-update"
+fi
+
 AC_SUBST(TOOLS)
 AC_MSG_RESULT($TOOLS)
 

